### PR TITLE
better reference sle 15 now

### PR DESCRIPTION
**Pull request by keichwa**

### CODE REVIEW

File: modules/reference/pages/systems/autoinst-distributions.adoc [hunk 1]
```diff
@@ -8,15 +8,15 @@ The [guimenu]``Distributions`` page enables you to find and create custom instal
...
+Installation data must be made available prior to creating a distribution, as described in the [ref]_SUSE Linux Enterprise Deployment Guide_ link:https://www.suse.com/documentation/sles-15/singlehtml/book_sle_deployment/book_sle_deployment.html#part.installserver[{sle} Deployment Guide] or, respectively, link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/chap-kickstart-installations[{rhel} Installation Guide : chap-kickstart-installations]
 This tree must be located in a local directory on the {productname} server.
 ====
 
 .Procedure: Creating a Distribution for Autoinstallation
 . To create a distribution, on the [guimenu]``Autoinstallable Distributions`` page click [guimenu]``Create Distribution`` in the upper right corner.
 . On the [guimenu]``Create Autoinstallable Distribution`` page, provide the following data:
 +
-** Enter a label (without spaces) in the [guimenu]``Distribution Label`` field, such as `my-orgs-sles-12-sp2` or ``my-orgs-rhel-as-7``.
+** Enter a label (without spaces) in the [guimenu]``Distribution Label`` field, such as `my-orgs-sles-15-sp1` or ``my-orgs-rhel-as-7``.
```
Comment (Loquacity) [inline modules/reference/pages/systems/autoinst-distributions.adoc]: And RHEL 8? ;)

Comment (mcalmer) [reply modules/reference/pages/systems/autoinst-distributions.adoc]: No, rhel8 is not yet supported

Comment (keichwa) [reply modules/reference/pages/systems/autoinst-distributions.adoc]: thanks for clarifying it!  I'm not familiar with RH.

Comment (Loquacity) [reply modules/reference/pages/systems/autoinst-distributions.adoc]: @mcalmer Is there an ETA on that?

Comment (mcalmer) [reply modules/reference/pages/systems/autoinst-distributions.adoc]: No, we only know it will be difficult. So I expect it may take longer.

**Pull request merged**